docs/core-concept.mdx

```diff
@@ -0,0 +1,76 @@
+---
+id: core-concept
+title: Core Concept
+---
+import {Image} from '@components'
+import { AdditionalInformation, Badge } from '@components';
+import ExternalLinkIcon from '@static/img/external-link.svg';
+
+<Image src="/img/core-concept-diagram.jpg"/>
+
+## 1. Service Packages Configuration
+The Drill4J Agent sends the application structure (packages\classes\methods) to the Drill admin panel. <br/>
+User is able to configure which service (application) packages will be instrumented. It can be done on the 2nd step of agent registration in the field **Application Packages**.
+<Image src="/img/agent-registration-step2.png"/>
+User has the ability to exclude unnecessary packages or classes via "!" before the package name. <br/>
+The System automatically changes ".", " " to "/". User doesn't have the ability to enter more then one "/" in a row. <br/>
+Line with hyphenation turns into an array of strings for backend.
+
+## 2. JaCoCo Instrumentation by Probes
+	
+Code is instrumented via JaCoCo. Instrumentation allows the Drill4j agent to track code execution during tests. 
+
+### JaCoCo Overview
+
+JaCoCo is a free code coverage library for Java, that calculates coverage by **probes**. <br/>
+**Probes** - boolean massive. <br/>
+For example: <br/>
+**[f,f,f,f,f,f,f.......]**, where **f** - false means that in the process, our thread of execution did not come across a line of code. This probes list exists for each class.
+Probes are placed in different places (e.g.):
+- at the beginning of the method
+- at the beginning of the branch block
+
+### Code Coverage Calculation Algorithm
+
+First, probes are put down with the help of instrumentation in the source code.
+When the code is executed, the probe becomes **TRUE** in the boolean massive.
+Analysis of all probes and coverage calculation is described in [documentation](https://www.jacoco.org/jacoco/trunk/index.html) <br/>
+Bytecode instrumentation is described in [documentation](https://www.baeldung.com/java-instrumentation) 
+	
+## 3. Context Transmission	
+	
+Autotest Agent tracks code execution for each test separately and reports these data to Drill4J Admin Backend. <br/> 
+It integrates with various [testing frameworks](/docs/supported-frameworks) and configures Selenium to add headers to the HTTP requests. <br/> 
```

Comment:
I suggest adding double line breaks (`<br/>`-s`), where it makes sense, **and** add no line breaks, where auto linewrap works. 
Otherwise, the text appeals a bit messy

![image](https://user-images.githubusercontent.com/34244611/165539511-06223d25-afe0-4839-81e1-abdca4734892.png)